File: public/static/templates/python/config/config_reader.py
import configparser
import argparse

def reader():
    config = configparser.ConfigParser()
    config.read('config/config.ini', encoding='utf-8')
    
    parser = argparse.ArgumentParser()
    parser.add_argument('--mode', '-m', type=str, default='Default')
    args = parser.parse_args()
    #print('args : ', type(args.mode))
    mode = args.mode
    
    
    
    return config[mode]

File: public/static/templates/go/config/config_reader.go
package config

import (
	"flag"
	"log"

	"github.com/alyu/configparser"
)

var Mode *string

func ConfigMode() {
	Mode = flag.String("platform", "Default", "select mode")
	flag.Parse()

}

func GetMode() string {
	return *Mode
}

func Reader(platform string) map[string]string {

	configparser.Delimiter = "="

	config, err := configparser.Read("config/config.ini")
	if err != nil {
		log.Fatal(err)
	}
	section, _ := config.Section(platform)
	options := section.Options()

	return options
}
